homeassistant/components/media_player/__init__.py

```diff
@@ -1187,12 +1187,10 @@ def get_browse_image_url(
         """Generate an url for a media browser image."""
         url_path = (
             f"/api/media_player_proxy/{self.entity_id}/browse_media"
-            # quote the media_content_id as it may contain url unsafe characters
```

Comment:
aiohttp fixed https://github.com/aio-libs/aiohttp/issues/5621 via https://github.com/aio-libs/aiohttp/pull/8898 so the urls are now decoded before they get to handler which means we can't pass quoted data to a handler via a regex anymore via this method.

I moved it to the query string instead